host_applications/linux/apps/gencmd/gencmd.c

```diff
@@ -36,6 +36,20 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "interface/vmcs_host/vc_vchi_gencmd.h"
 
+void show_usage()
+{
+   printf( "Usage: vcgencmd [-t] command\n" );
+   printf( "Send a command to the VideoCore and print the result.\n\n" );
+   printf( "  -t         Time how long the command takes to complete\n" );
+   printf( "Use the command `vcgencmd commands` to get a list of available commands\n\n" );
+   printf( "Exit status:\n" );
+   printf( "   0    command completed successfully\n" );
+   printf( "  -1    problem with VCHI\n" );
+   printf( "  -2    VideoCore returned an error\n\n" );
+   printf( "For further documentation please see\n" );
+   printf( "raspberrypi.org/documentation/raspbian/applications/vcgencmd.md\n\n" );
```

Comment:
Is it common practice to put URLs in help strings? :man_shrugging: 
In any case, it should probably be `https://www.rasperrypi.org/...` ?

Reply:
No idea. Since there is no man page, and the command is fully documented on the website, I've added a link to it. I don't really care if it is not common practice.